src/p2f/src/CodeOwls.PowerShell/CodeOwls.PowerShell.Provider/Provider.cs

```diff
@@ -54,6 +56,34 @@ public abstract class Provider : NavigationCmdletProvider,
         IContentCmdletProvider
     {
         private static readonly Dictionary<string, Regex> FilterRegexMap = new Dictionary<string, Regex>(StringComparer.OrdinalIgnoreCase);
+        private static readonly string StringSeparator = (IsWindows) ? "\\" : "/";
+        private static readonly char CharSeparator = (IsWindows) ? '\\' : '/';
+        private static bool? _isWindows = null;
+
+        /// <summary>
+        /// True if the current platform is Windows.
+        /// </summary>
+        public static bool IsWindows
```

Comment:
This shouldn't be necessary if you use the cross-platform .Net APIs

Reply:
Yes no need